Bottom: 39dcc20455593bdd052c29f0906f97b98602ae37
Top:    a493f510ad9ba75e05962bdbacaad61f4f8d2909
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 18:51:23 +0000

Revert "Just make a black background."

This reverts commit cf6c95ade3be149a5a8659bd13a80eedc4092fd2.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 8de6c63..d623df8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2090,9 +2090,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glEnable (GL_STENCIL_TEST);
   glEnable (GL_DEPTH_TEST);
   glDepthFunc (GL_ALWAYS);
-  glClearColor (port->bg_color.red / 65535.,
-                port->bg_color.green / 65535.,
-                port->bg_color.blue / 65535.,
+  glClearColor (port->offlimits_color.red / 65535.,
+                port->offlimits_color.green / 65535.,
+                port->offlimits_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2196,7 +2196,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
   glDepthMask (GL_FALSE);
-#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2227,7 +2226,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
-#endif
 
   glDepthMask (GL_TRUE);
